lib/tasks/calibration_demo.rake

```
@@ -0,0 +1,244 @@
+
+    assignment = Assignment.find_by(id: 8) || Assignment.find_by(name: 'Calibration Demo Assignment')
+    assignment ||= Assignment.create!(
+      name: 'Calibration Demo Assignment',
+      instructor: instructor,
+      has_teams: true,
+      private: false
+    )
+    assignment.update!(instructor: instructor, has_teams: true, private: false)
+
+    reviewee_team =
+      if assignment.id == 8 && ReviewResponseMap.exists?(id: 8, reviewed_object_id: assignment.id, for_calibration: true)
```

RuboCop (1.86.1)

[convention] Line is too long. [121/120]

(Layout/LineLength)